go/pools/smartconnpool/pool.go

```diff
@@ -36,6 +36,12 @@ var (
 
 	// ErrCtxTimeout is returned if a ctx is already expired by the time the connection pool is used
 	ErrCtxTimeout = vterrors.New(vtrpcpb.Code_DEADLINE_EXCEEDED, "resource pool context already expired")
+
+	// ErrShutdown is returned when trying to get a connection from a closed conn pool
+	ErrConnPoolClosed = vterrors.New(vtrpcpb.Code_INTERNAL, "connection pool is closed")
```

Comment:
ErrCtxTimeout (line 38) has "resource pool" in the error message. This one has "connection pool". Should they be consistent?